cartographer_ros/cartographer_ros/map_builder_bridge.h

```diff
@@ -44,32 +44,33 @@ class MapBuilderBridge {
     TrajectoryOptions trajectory_options;
   };
 
-  MapBuilderBridge(const NodeOptions& node_options, tf2_ros::Buffer* tf_buffer);
+  MapBuilderBridge(const NodeOptions &node_options, tf2_ros::Buffer *tf_buffer);
```

Comment:
Yes, please. Re-running `clang-format` cannot fix this issue since it changes everything in the file to match the majority style. And right now the majority of cases in this file are clearly different from the rest of the code base.